### PR TITLE
fix: Lead one styling issue

### PR DESCRIPTION
**Pull request by aronshamash**
- Fixes styling issues on `LeadOneFront`

### CODE REVIEW

**Review by Geoff-Ford — APPROVED** [review]

File: packages/edition-slices/src/slices/frontleadone/index.js [hunk 1]
```diff
@@ -38,8 +38,8 @@ function renderMedium(props, breakpoint, orientation) {
 }
 
 const FrontLeadOne = (props) => {
-  const renderSlice = (breakpoint, orientation) =>
-    renderMedium(props, breakpoint, orientation);
+  const renderSlice = (_breakpoint, orientation) =>
+    renderMedium(props, orientation);
```
Comment (c-j-j) [inline packages/edition-slices/src/slices/frontleadone/index.js:42]: Not sure how this fixes it... but `renderMedium` takes in 3 args, with the second being `breakpoint`.

Comment (aronshamash) [reply packages/edition-slices/src/slices/frontleadone/index.js:40]: sorry missed some other changes to remove breakpoint from tile. see next commit

Comment (c-j-j) [reply packages/edition-slices/src/slices/frontleadone/index.js:40]: Having said that, `breakpoint` isn't needed for frontleadone, so can be removed

Comment (Geoff-Ford) [reply packages/edition-slices/src/slices/frontleadone/index.js:40]: You are right, it is still an argument in `renderMedium` and also passed to one of the tiles

**Review by c-j-j — APPROVED** [review]

**Pull request merged**